dbms/src/Storages/DeltaMerge/DMSegmentThreadInputStream.h

```diff
@@ -106,7 +106,11 @@ class DMSegmentThreadInputStream : public IProfilingBlockInputStream
             }
             FAIL_POINT_PAUSE(FailPoints::pause_when_reading_from_dt_stream);
 
+            Stopwatch watch_read;
             Block res = cur_stream->read(res_filter, return_filter);
+            auto elapsed = watch_read.elapsedSeconds();
+            GET_METRIC(tiflash_storage_subtask_count, type_table_scan).Increment();
+            GET_METRIC(tiflash_storage_subtask_duration_seconds, type_table_scan).Observe(elapsed);
```

Comment:
`tiflash_storage_subtask` intend for some compaction tasks/index update tasks.

Reading a block from the storage layer is quite different from those tasks. Maybe add new metrics for it?

Reply:
fixed